src/org/parosproxy/paros/CommandLine.java

```diff
@@ -328,6 +332,28 @@ private boolean parseKeywords(String[] args, int i) throws Exception {
                 this.configs.put(pair.substring(0, eqIndex), pair.substring(eqIndex + 1));
                 result = true;
             }
+        } else if (checkPair(args, CONFIG_FILE, i)) {
+            String conf = keywords.get(CONFIG_FILE);
+            File confFile = new File(conf);
+            if (! confFile.exists()) {
+            	// We cant use i18n here as the messages wont have been loaded
+            	String error = "No such file: " + confFile.getAbsolutePath();
+            	System.out.println(error);
+                throw new Exception(error);
+            } else if (! confFile.canRead()) {
+            	// We cant use i18n here as the messages wont have been loaded
+            	String error = "File not readable: " + confFile.getAbsolutePath();
+            	System.out.println(error);
+                throw new Exception(error);
+            }
+            Properties prop = new Properties();
+            try (FileInputStream inStream = new FileInputStream(confFile)) {
+    			prop.load(inStream);

```

Comment:
In the meantime new code should fit in with existing code. Which I keep on getting wrong :/
We should have done a global replace when we released 2.5.0. Guess that should be done for 2.6.0 now...